Trimble.Modus.Components/Controls/TabBar/TMTabbedPage.cs

```diff
@@ -0,0 +1,347 @@
+﻿using System.Collections;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using Trimble.Modus.Components.Constant;
+using Trimble.Modus.Components.Helpers;
+
+namespace Trimble.Modus.Components;
+
+[ContentProperty(nameof(TabItems))]
+public partial class TMTabbedPage : ContentPage
+{
+    #region Private Fields
+    private Grid mainContainer;
+    private Grid tabStripContainer;
+    private CarouselView contentContainer;
+    private List<double> contentWidthCollection;
+    private ObservableCollection<TabViewItem>? contentTabItems;
```

Comment:
Why do we need to two arrays of TabViewItem?

Reply:
the tab with contents are only added towards the strip which is contentTabItems, TabItems From the user contains all the tabs